multigeiger/multigeiger.ino

```diff
@@ -308,92 +307,92 @@ void setup()
   pinMode      (DISPLAY_ON, OUTPUT);
   digitalWrite(DISPLAY_ON,HIGH);
 #endif
-  // Initialize Pins
+  // initialize pins
   digitalWrite (PIN_SPEAKER_OUTPUT_P, LOW);
-  digitalWrite (PIN_SPEAKER_OUTPUT_N, HIGH); 
+  digitalWrite (PIN_SPEAKER_OUTPUT_N, HIGH);
 
-  // set Interrupts (on pin change), attach interrupt handler
+  // set interrupts (on pin change), attach interrupt handler
   attachInterrupt (digitalPinToInterrupt (PIN_HV_CAP_FULL_INPUT), isr_GMZ_capacitor_full, RISING);// Capacitor full
   attachInterrupt (digitalPinToInterrupt (PIN_GMZ_count_INPUT), isr_GMZ_count, FALLING);          // GMZ pulse detected
 
-  // set and init Serial Communication 
+  // set and init serial communication
   if (Serial_Print_Mode != Serial_None) {
     noInterrupts();
     Serial.begin(115200);
     while (!Serial) {};
   }
-  
-  Serial.printf("Los Gehts! \n");
+
+  Serial.printf("Los Geht's! \n");
   uint32_t xx = getESPchipID();
 
   // build SSID
   sprintf(ssid,"ESP32-%d",xx);
-  
-  // Check, if we have an BME280 connected:
+
+  // Check if we have a BME280 connected:
   haveBME280 = bme.begin();
-//  Serial.printf("BME_Status: %d  ID:%0X\n", haveBME280, bme.sensorID());
+  //  Serial.printf("BME_Status: %d  ID:%0X\n", haveBME280, bme.sensorID());
 
-  // Setup IoTWebConf
+  // Set up IoTWebConf
   iotWebConf.setConfigSavedCallback(&configSaved);
   iotWebConf.setThingName(ssid);
   iotWebConf.init();
 
 
   // -- Set up required URL handlers on the web server.
   server.on("/", handleRoot);
-  server.on("/config", []{ iotWebConf.handleConfig(); });
-  server.onNotFound([](){ iotWebConf.handleNotFound(); });
+  server.on("/config", []{ iotWebConf.handleConfig(); });  // XXX WTF is that?
+  server.onNotFound([](){ iotWebConf.handleNotFound(); }); // XXX WTF is that?
```

Comment:
can somebody explain?

i mostly do Python code these days, so i am wondering whether that is valid C?